Documentation/ApiOverview/BackendModules/TemplateClass/Index.rst

```diff
@@ -15,6 +15,12 @@ The Backend Template View
    this manual if you need a reference to the old way of programming
    backend modules.
 
+.. tip::
+
+   If you don't want to do extensive data modelling consider working without
+   :ref:`Extbase templating. <backend-modules-template-without-extbase>`
+
+
 Modern backend modules are written using the Extbase/Fluid combination.
```

Comment:
We should probably adjust this sentence to be less "absolute" (maybe "can be written")

Reply:
Tried to be less "absolute"

Thanks for reviewing